packages/command/pubspec.yaml

```diff
@@ -7,7 +7,7 @@ documentation: https://github.com/FlutterGen/flutter_gen
 issue_tracker: https://github.com/FlutterGen/flutter_gen/issues
 
 environment:
-  sdk: '>=2.17.0 <4.0.0'
+  sdk: '>=3.0.0'
```

Comment:
Could you use `^3.0.0`? I believe `dart pub publish --dry-run` will complain about this

Reply:
I fixed! [1b0beff](https://github.com/FlutterGen/flutter_gen/pull/658/commits/1b0beff687afeed6c152f439fe3a85f6390d7a93)